### ISSUE TITLE
ignore_errors=true doesn't read any row but ignore_errors=false does.

### ISSUE THREAD

**Issue by tanin47** [duckdb/duckdb#10841]
### What happens?

I have this CSV:

```
"first_name",last_name,"email"
john,"doe, do", "test@doe.com"
"nanakorn, "" tanin", somename ,some email
```

Here's how duckdb 0.10.0 behaves:

```sql
select * from read_csv('quote.csv', delim=',', header = true,  all_varchar = true, null_padding = true, ignore_errors = true);
```
```
┌──────────────┬───────────┬─────────┐
│ "first_name" │ last_name │ "email" │
│   varchar    │  varchar  │ varchar │
├────────────────────────────────────┤
│               0 rows               │
└────────────────────────────────────┘
```
```sql
select * from read_csv('quote.csv', delim=',', header = true,  all_varchar = true, null_padding = true);
```
```
┌───────────────────┬────────────┬──────────────┐
│    first_name     │ last_name  │    email     │
│      varchar      │  varchar   │   varchar    │
├───────────────────┼────────────┼──────────────┤
│ john              │ doe, do    │ test@doe.com │
│ nanakorn, " tanin │  somename  │ some email   │
└───────────────────┴────────────┴──────────────┘
```

### To Reproduce

You can use the CSV and run the SQL as shown above

### OS:

Mac

### DuckDB Version:

0.10.0

### DuckDB Client:

Node

### Full Name:

Tanin Na Nakorn

### Affiliation:

None

### Have you tried this on the latest [nightly build](https://duckdb.org/docs/installation/?version=main)?

I have tested with a release build (and could not test with a nightly build)

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] Yes, I have

**Comment by szarnyasg** [comment]
This is a duplicate of #10769, which was fixed by #10777. If you compile DuckDB from `main`, it will work. The next release is scheduled to [mid-March](https://duckdb.org/dev/release-calendar).

**Comment by tanin47** [comment]
Thank you!
